client/my-sites/checkout/composite-checkout/components/wp-line-item.tsx

```diff
@@ -222,19 +200,25 @@ function WPNonProductLineItem( {
 			<span aria-labelledby={ itemSpanId } className="checkout-line-item__price">
 				<LineItemPrice actualAmount={ actualAmountDisplay } isSummary={ isSummary } />
 			</span>
-			{ hasDeleteButton && removeProductFromCart && formStatus === FormStatus.READY && (
+			{ hasDeleteButton && removeProductFromCart && (
 				<>
-					<DeleteButton
-						coupon={ coupon }
-						className="checkout-line-item__remove-product"
-						buttonType="borderless"
-						disabled={ isDisabled }
-						onClick={ () => {
-							setIsModalVisible( true );
-						} }
-					>
-						<DeleteIcon uniqueID={ deleteButtonId } product={ label } />
-					</DeleteButton>
+					<DeleteButtonWrapper>
+						<DeleteButton
+							className="checkout-line-item__remove-product"
+							buttonType={ 'text-button' }
+							aria-label={ String(
+								translate( 'Remove %s from cart', {
+									args: label,
+								} )
+							) }
+							disabled={ isDisabled }
+							onClick={ () => {
+								setIsModalVisible( true );
+							} }
+						>
+							{ translate( 'Remove from cart' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Remove from Cart' )` *ES Score: 9*